### PR TITLE
km create accessing atlases guide

### PR DESCRIPTION
**Pull request by kayleemathews**
I created a new guide for the HCA Data Portal describing how to access atlas data from the Data Explorer. I also made a few minor changes to the existing guides, mainly to reflect the current state of the Portal and Explorer. 

@NoopDog and @ekiernan would you mind taking a look?

### CODE REVIEW

File: content/guides/userguides/quick-start-guide.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ After you identify a cohort of interest, you download the raw data, analysis fil
 
 This will open a new page giving you the option to:
 
-1) Download Selected Data Using "curl"
+1) Download Study Data and Metadata (Curl Command)
 2) Download a File Manifest with Metadata for the Selected Data
-3) Export to Terra 
+3) Export Study Data and Metadata to Terra Workspace
```
Comment (ekiernan) [inline content/guides/userguides/quick-start-guide.md]:
```suggestion
3) Export Study Data and Metadata to a Terra Workspace

File: content/guides/userguides/quick-start-guide.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ This metadata, also called a "Data Manifest" is in TSV file format and lists all
 
 To download the metadata from the **Export Selected Data** page:
 
-1. Go to **Download a File Manifest with Metadata for the Selected Data** and select **Start**
+1. Under **Download a File Manifest with Metadata for the Selected Data** and select **Request File Manifest**
```
Comment (ekiernan) [inline content/guides/userguides/quick-start-guide.md]:
```suggestion
1. Under **Download a File Manifest with Metadata for the Selected Data**, select **Request File Manifest**

**Review by ekiernan — APPROVED** [review]
A few minor updates. The new docs look great and thank you for improving the old ones.

File: content/guides/userguides/quick-start-guide.md (hunk 2 above)
Comment (ekiernan) [reply content/guides/userguides/quick-start-guide.md]: @kayleemathews this does not seem to make sense the way it's written currently:

 "1. Under **Download a File Manifest with Metadata for the Selected Data** and select **Request File Manifest**

**Review by NoopDog — APPROVED** [review]

**Pull request merged**